CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Contributing
+
+We are always welcome your contribution!
+
+## How to contribute
+
+### Find the tasks
+
+We use GitHub issues to manage the tasks. Please find the issues you'd like to contribute to it. `welcome contribute` and `easy` are good for the first contribution.
+
+Of course, it would be great to send PullRequest which has no issue! However, please explain the motivation, the cause, the solution and so on in the description of the PR then.
+
+### Start contributing
+
+If you find the tasks you want to contribute, please comment on the issue to prevent to conflict contribution. We'll reply as soon as possible, but it's unnecessary to wait for our reaction if you have raised voice. It's okay to start contribution and send PullRequest!
```

Comment:
If you find a task you want to work on, please comment on the issue to prevent conflicts. We'll reply as soon as possible, but you don't need to wait for our reaction, you can start working on the task once you have commented on the issue!